PyInstaller/loader/pyimod03_importers.py

```diff
@@ -491,6 +490,123 @@ def get_filename(self, fullname):
                 fullname.replace('.', pyi_os_path.os_sep) + '.pyc')
         return filename
 
+    def find_spec(self, fullname, path=None, target=None):
+        """
+        PEP-451 finder.find_spec() method for the ``sys.meta_path`` hook.
+
+        fullname     fully qualified name of the module
+        path         None for a top-level module, or package.__path__ for submodules or subpackages.
+
+        Finders must return ModuleSpec objects when find_spec() is called. This new method replaces find_module()
+        and find_loader() (in the PathEntryFinder case). If a loader does not have find_spec(), find_module() and
+        find_loader() are used instead, for backward-compatibility.
+        """
+        # Acquire the interpreter's import lock for the current thread. This
+        # lock should be used by import hooks to ensure thread-safety when
+        # importing modules.
```

Comment:
The comment says "Acquire … lock", which is not done here. Please correct the comment, it is at least confusing if not wrong.